cobrawap/pipeline/utils/Snakefile

```diff
@@ -12,11 +12,13 @@ from utils.snakefile import get_setting
 CONFIG_PATH = Path(get_setting('config_path'))
 OUTPUT_PATH = Path(get_setting('output_path'))
 
-if 'STAGE_INPUT' not in config:
-    logger.warning('No STAGE_INPUT defined for running the stage individually! '
-                   'You can set it via the command line with '
-                   '`--config STAGE_INPUT=/path/to/file`.')
+is_first_stage = config['STAGE_NAME'] == get_setting('stages')['1']
+if 'STAGE_INPUT' not in config or ('STAGE_INPUT' in config and is_first_stage):
     config['STAGE_INPUT'] = None
+    if not is_first_stage:
+        logger.warning('No STAGE_INPUT defined for running stage \'{}\' individually! '.format(config['STAGE_NAME']) +
+                       'You can set it via the command line with ' +
+                       '`--config STAGE_INPUT=/path/to/file`.')
```

Comment:
This logic got a bit messy, how about we simplify it like this:

```suggestion
if is_first_stage:
    config['STAGE_INPUT'] = None
if 'STAGE_INPUT' not in config:
    logger.warning('No STAGE_INPUT defined for running stage \'{}\' individually! '.format(config['STAGE_NAME']) +
                       'You can set it via the command line with ' +
                       '`--config STAGE_INPUT=/path/to/file`.')
```

Reply:
I suggest to accept the change.
I also suggest to consider to send an alert to the user any time (as in this case) when the user's settings have been overwritten, something like: logger.warning(Cobrawap is running from first stage - ‘STAGE_INPUT forced to Nan. Your user settings have been overwritten)